tools/bin/sdc-sbcreate

```diff
@@ -55,22 +57,22 @@ EOF
 
 function fatal
 {
-    echo "`basename $0`: $*"
+    echo "$(basename "$0"): $*"
     exit 1
 }
 
 function do_countdown
 {
     echo ""
-    echo "This tool will create an SDC support bundle in `dirname $TEMPDIR`."
+    echo "This tool will create a Triton support bundle in $(dirname "${TEMPDIR:-/tmp}")."
     echo "This may take some time...  If you wish to abort press CTRL-C now!"
     echo ""
-    countdown=$((${COUNTDOWN} - 1))
-    while [[ ${countdown} -ge 0 ]]; do
+    countdown=$(( COUNTDOWN - 1))
+    while (( countdown > 0 )); do
```

Comment:
Not a blocker, but is this (( )) syntax a bash-ism?  Or a POSIX shell-ism?

Reply:
I believe it's a bashism, but this script is bash.